examples/ldap-add-remove-entry/ldap_add_remove_entry.bal

```diff
@@ -0,0 +1,27 @@
+import ballerina/io;
+import ballerina/ldap;
+
+public function main() returns error? {
+    // Initializes a new LDAP client with credentials.
+    ldap:Client ldapClient = check new ({
```

Comment:
Refer to this comment [1]

[1] - https://github.com/ballerina-platform/ballerina-distribution/pull/5578/files#r1718020943